modules/ROOT/pages/afm-monitoring-private-apis.adoc

```diff
@@ -18,30 +18,44 @@ All incoming HTTP traffic must be allowed into the environment where a private l
+image::afm-monitor-private-2.png[]
++
+A monitor runs on two separate schedules, and each is associated with the same private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in a larger number of private APIs. The private locations and both APIs are all in a single environment.
 
-* Second relationship type (represented by dashed green arrows)
+* Third example configuration
++
+image::afm-monitor-private-3.png[]
 +
-Monitor 2 runs on two separate schedules, and each is associated with the same private location. The private location is shared with the schedule that runs Monitor 1. Monitor 2 is written to test one or more endpoints in private API 3 and in private API 4, though it could also test endpoints in more private APIs. The private locations and both APIs are all in a single environment.
+A monitor runs on two separate schedules, and each is associated with a different private location. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in a larger number of private APIs.
 
-* Third relationship type (represented by dashed-and-dotted blue arrows)
+* Fourth example configuration
 +
-Monitor 3 runs on two separate schedules, and each is associated with a different private location. Monitor 3 is written to test one or more endpoints in API 5 and in API 6, though it could also test endpoints in more private APIs.
+image::afm-monitor-private-4.png[]
++
+A monitor runs on two separate schedules, and each is associated with a different private location. Each private location is in a separate environment. One environment could be a test environment, and the other environment could be a production environment. Private API 1 and Private API 2 are copies of the same API. The monitor is written to test one or more endpoints in private API 1 and the same endpoints in API 2, though it could also test endpoints in a larger number of private APIs.
 
-* Fourth relationship type (represented by dotted orange arrows)
+* Fifth example configuration
++
+image::afm-monitor-private-5.png[]
 +
-Monitor 4 runs on two separate schedules, and each is associated with a different private location. Each private location is in a separate environment. One environment could be a test environment, and the other environment could be a production environment. Private API 7 and Private API 8 are copies of the same API. Monitor 4 is written to test one or more endpoints in private API 7 and the same endpoints in API 8, though it could also test endpoints in more private APIs.
+A monitor runs on one schedule, though it could run on more than one. The schedule is associated with a private location. The monitor is written to test one or more endpoints in public API, which is available on the world-wide web, 1 and in private API 2, though it could also test endpoints in a larger number of APIs.
```

Comment:
"in public API" = which public API, or did you mean just "a" public API?
Oh...I think you mean...:
```suggestion
A monitor runs on one schedule, although it could run on more than one. The schedule is associated with a private location. The monitor is written to test one or more endpoints in public API 1, which is available on the web, and in private API 2, although it could also test endpoints in a larger number of APIs.
```
Since we are providing several specific situations, could it be confusing to audiences (or simply not necessary) to mention the "although"s?